Formula/gauge.rb

```diff
@@ -3,8 +3,8 @@
 class Gauge < Formula
   desc "Test automation tool that supports executable documentation"
   homepage "http://getgauge.io"
-  url "https://github.com/getgauge/gauge/archive/v0.7.0.tar.gz"
-  sha256 "959ad3feea738d107e8840804b8faf82d1c1242db8beb1f746e21c76d7277f09"
+  url "https://github.com/getgauge/gauge/archive/v0.8.0.1.tar.gz"
```

Comment:
Please update the version numbers in your PR (commit message and PR title) to accurately reflect the exact version being built.